orc8r/cloud/go/obsidian/swagger/handlers/handlers.go

```diff
@@ -14,17 +14,84 @@
 package handlers
 
 import (
+	"bytes"
+	"html/template"
 	"net/http"
+	"strings"
 
 	"magma/orc8r/cloud/go/obsidian"
 	"magma/orc8r/cloud/go/obsidian/swagger"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/lib/go/registry"
 
 	"github.com/labstack/echo"
+	"github.com/labstack/echo/middleware"
+	"github.com/pkg/errors"
 )
 
-// GetGenerateCombinedSpecHandler returns a routing handler which creates
+// Defines the struct to be templated into Swagger UI.
```

Comment:
"UIInfo contains the templating variables injected into the Swagger UI template."